plugin/vcs/vcs.go

```diff
@@ -117,6 +125,12 @@ type Provider interface {
 	// repositoryID: the repository ID from the external VCS system (note this is NOT the ID of Bytebase's own repository resource)
 	FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx common.OauthContext, instanceURL string, repositoryID string) ([]*RepositoryMember, error)
 
+	// Fetch all repository list within the scope of the given user
+	//
+	// oauthCtx: OAuth context to write the file content
+	// instanceURL: VCS instance URL
+	FetchRepositoryList(ctx context.Context, oauthCtx common.OauthContext, instanceURL string) ([]byte, error)
```

Comment:
We should define a Repository similar to the RepositoryMember defined above, and let the caller to convert it later.

Otherwise, we are breaking the encapsulation. If somehow we need to change the format in the returned []byte, we will make changes all the way up to the frontend code.